versioned_docs/version-V3/concepts/introduction/liquidity-user-guide.md

```diff
@@ -55,9 +55,9 @@ Please note that selecting a starting price that differs significantly from the
 
 LPs must select a price range in which to provide liquidity.
 
-Depending on the fee tier selected, the interface will offer the ability to adjust prices in either 0.10%, 0.60%, or 2.00% increments. Manually entered prices will automatically adjust to the closest associated price tick.
+Depending on the fee tier selected (0.05% / 0.30% / 1.00% fee), the interface will offer the ability to adjust prices in either 0.10%, 0.60%, or 2.00% price tick increments. Manually entered prices will automatically adjust to the closest associated price tick.
```

Comment:
`price tick` doesn't make sense here - `price` is better

Reply:
Edited as suggested. Thanks.